### PR TITLE
[pack]Remove duplicate workers directories from V3 site extension

### PR DESCRIPTION
**Pull request by pragnagopa**
Reduces site extension size by ~176MB

### CODE REVIEW

**Comment by pragnagopa** [comment]
@brettsam - should we consider taking this change into current release?

File: src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs [hunk 1]
```diff
@@ -33,6 +33,11 @@ public RpcWorkerConfigFactory(IConfiguration config, ILogger logger, ISystemRunt
             _environment = environment ?? throw new ArgumentNullException(nameof(environment));
             _metricsLogger = metricsLogger;
             WorkersDirPath = Path.Combine(Path.GetDirectoryName(new Uri(typeof(RpcWorkerConfigFactory).Assembly.CodeBase).LocalPath), RpcWorkerConstants.DefaultWorkersDirectoryName);
+            if (!Directory.Exists(WorkersDirPath))
+            {
+                // Site Extension. Default to parent directory
+                WorkersDirPath = Path.Combine(Directory.GetParent(Path.GetDirectoryName(new Uri(typeof(RpcWorkerConfigFactory).Assembly.CodeBase).LocalPath)).FullName, RpcWorkerConstants.DefaultWorkersDirectoryName);
```
Comment (brettsam) [inline src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs]: Same comment in the other one about reusing `Path.GetDirectoryName(new Uri(typeof(RpcWorkerConfigFactory).Assembly.CodeBase).LocalPath)`

**Comment by brettsam** [comment]
Yeah we should probably take this -- that's a significant size improvement.

**Comment by pragnagopa** [comment]
Closing in favor of https://github.com/Azure/azure-functions-host/pull/5725

**Pull request closed without merging**